components/AboutAiesec.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+    <div>
+        <h2 class="About-aiesec">About AIESEC</h2>
+        <div class="Explain">
+            <p>AIESEC(アイセック)は、海外ボランティアや海外ボランティアプログラムなどの</p>
+            <p>主観事業を通して、世界中の若者のリーダーシップを育むことを目指している</p>
+            <p>グローバルプラットフォームです。1948年にヨーロッパで創設されて以来、</p>
+            <p>「平和で、人々の可能性が最大限発揮された社会」の実現を目指して活動しています。</p>
+        </div>
+        <div class="content">
+            <img src="" alt="">
+            <p>114の国と地域</p>
+            <p>の支部</p>
+        </div>
+        <div class="content">
+            <img src="" alt="">
+            <p>7000以上の</p>
+            <p>パートナー団体</p>
+        </div>
+        <div class="content">
+            <img src="" alt="">
+            <p>年間30000件以上の</p>
+            <p>経験を提供</p>
+        </div>
+    </div>
+</template>
+
+<script>
+</script>
+
+<style scoped lang="scss">
+.About-aiesec{
+    color: #037Ef3;
+    font-size: 24px;
+    padding: 20px;
+    letter-spacing: 2px;
+    font-family: "Lato";
+}
+.Explain{
+    font-size: 16px;
+    letter-spacing: 2px;
+    font-family: "游ゴシック";
+}
+.content{
+    float: left;
+    margin-right: 40px;
```

Comment:
margin-rightだと最後の要素にも右に40pxのmarginができてしまい、
全体で見ると若干左に寄ってしまうので左右にmarginを入れてください

```suggestion
    margin: 0 40px;
```